saga/sagalogs/memory_test.go

```diff
@@ -119,3 +121,54 @@ func TestMemorySagaGC(t *testing.T) {
 		t.Fatalf("Unexpected nil error logging message to GC'd saga")
 	}
 }
+
+/*
+	Set different saga update channel size using DefaultSagaUpdateChSize
+	Benchmarked on cpu: Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
+
+	For sagasCount = 10, updatesPerSaga = 10000 and following DefaultSagaUpdateChSize values
+	* DefaultSagaUpdateChSize=1:     1 iteration(s)	124623537403 ns/op
+	* DefaultSagaUpdateChSize=10:	 1 iteration(s)  12299526501 ns/op
+	* DefaultSagaUpdateChSize=100:	 1 iteration(s)	  1864785416 ns/op
+	* DefaultSagaUpdateChSize=1000:	 3 iteration(s)	   451137271 ns/op
+	* DefaultSagaUpdateChSize=10000: 7 iteration(s)	   176294414 ns/op
```

Comment:
Hopefully gives a big boost to utilization!

Reply:
Yeah, fingers crossed! Might have to tune the DefaultSagaUpdateChSize or the limit on the number of updates processed together and see how it impacts the performance in real-time.